app/Amortization/Amortization.php

```diff
@@ -110,6 +136,39 @@ private function getSuperLoaPaymentPlans()
         }
         return $plan;
     }
+    private function getNoBsNewPaymentPlans()
```

Comment:
`getNoBsNewPaymentPlans` and `getNoBsRenewalPaymentPlans` looks similar, can't we reuse logic?